packages/api/src/external/hie/__tests__/get-hie-initiator.test.ts

```diff
@@ -61,23 +61,6 @@ describe("getHieInitiator", () => {
     expect(resp.name).toBe(facility.data.name);
   });
 
-  it("returns the organization as initiator when is not OBO", async () => {
```

Comment:
It feels like this test should be replaced by the situation that led to this PR.

Like, at least this should be good to have unit tests representing the possible combinations, right?

```ts
if (
    isOboEnabled(facility, hie) ||
    (isCandidateImplementor(organization.type) && isNonOboFacility(facility.type))
  ) {
```